### ISSUE TITLE
The image should be a hard drive image, not ISO9660

### ISSUE THREAD

**Issue by AkihiroSuda** [lima-vm/alpine-lima#123]
All the official Lima templates except Alpine uses hard drive images, and it is hard to maintain ISO booting specifically for Alpine

**Comment by afbjorklund** [comment]
Being able to use a "LiveCD" ISO with Lima is a great feature (since the docker-machine days), and U hope that it is never removed from Lima...

**Comment by AkihiroSuda** [comment]
The ISO build can continue to be supported, but probably not as a "Tier 1"

**Comment by AkihiroSuda** [comment]
> > The ISO build can continue to be supported, but probably not as a "Tier 1"
> 
> That just means you want to remove the ⭐ on the Alpine entry on https://lima-vm.io/docs/templates/, or anything else? I'm fine with removing the ⭐.

Yes, inclined to demote Alpine from Tier 1 ⭐ to Tier 2 ☆

**Comment by jandubois** [comment]
As for creating a hard drive image for Alpine, are there any tools that would convert an ISO into a disk image? If this is easy to do, then I'm happy to add it.

But if it is a completely separate process to build, then I'm personally not interested and would wait for somebody else to implement it.

**Comment by afbjorklund** [comment]
When I made my distributions, it was separate build targets. Same rootfs, but different imaging tools at the end...
It was Buildroot rather than Alpine, but I assume that it is the same underlying `mkisofs` and `mkfs.ext4` programs?

I used the LiveCD for [containers](https://github.com/afbjorklund/buildroot4containers) (same as in docker-machine, and here), but then used resizable HDD for [kubernetes](https://github.com/afbjorklund/buildroot4kubernetes).
The main other difference there was changing to from openrc/musl to systemd/glibc, which made it much bigger.
